core/tests/protractor_utils/LibraryPage.js

```diff
@@ -81,6 +81,19 @@ var LibraryPage = function() {
     return waitFor.pageToFullyLoad();
   };
 
+  this.addToPlaylist = function() {
```

Comment:
Make this `addExplorationToPlaylist. (as the module name is `LibraryPage` and library page contains other things as well.)

Reply:
or `addSelectedExplorationToPlaylist` as this method isn't accepting any parameter related to `which exploration`.

Reply:
Done